glotaran/builtin/megacomplexes/coherent_artifact/coherent_artifact_megacomplex.py

```diff
@@ -73,8 +73,11 @@ def finalize_data(
             global_dimension = dataset_model.get_global_dimension()
             model_dimension = dataset_model.get_model_dimension()
             dataset.coords["coherent_artifact_order"] = np.arange(1, self.order + 1)
+            concentration_dimensions = (model_dimension, "coherent_artifact_order")
```

Comment:
nitpicky, but don't call it "concentration". A coherent artifact is basically backreflections etc introduced by sample container. Call it "coherent_artifact_dimensions" or just "dimensions"

Reply:
The reason I called it that way is because those are the dimensions for `coherent_artifact_concentration` (line 79).
@joernweissenborn  @jsnel Should we maybe rename `coherent_artifact_concentration`?

Reply:
Hmm, this was my mistake naming it like that. "Amplitude" would be the right name. Could change the dataset item from `coherent_artifact_dimension` to `coherent_artifact_amplitude` in this PR?

Reply:
Agree that 'concentration' is not the right terminology here. Amplitudes is a bit ambiguous (could refer to both the amplitude as function of time at a given wavelength, or the (spectral) amplitude (clp) at a given wavelength). 
Maybe coherent_artifact_response is more appropriate here, since that's a commonly used term. See e.g.: 
https://link.springer.com/article/10.1134/1.1884668